### PR TITLE
San 25 be able to delete products as an admin

### PR DESCRIPTION
**Pull request by vikiorf**

### CODE REVIEW

File: backend/src/controllers/auth.controller.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ export const resetDatabaseController = async (req: Request, res: Response) => {
 
   try {
     await initAdminUser();
-    await initProducts();
+    await initProducts(true);
```
Comment (Markussim) [inline backend/src/controllers/auth.controller.ts:148]: Why? I don't see any arguments defined for the function

**Review by Markussim — APPROVED** [review]
LGTM, but see comment

File: backend/src/controllers/auth.controller.ts (hunk 1 above)
Comment (vikiorf) [reply backend/src/controllers/auth.controller.ts:148]: An optional boolean is defined for the function. See [here](https://github.com/iceberry-code/scar/pull/22/commits/a3ea1c17f19b384ec02ab25588e0b139b554d99e#diff-fd0281d10b0817dd7d5a4012e6beed85bacefb0cf029dc82b4d831f936359309R10)

Comment (Markussim) [reply backend/src/controllers/auth.controller.ts:148]: I read the old code. I am stupid :sunglasses:

**Pull request merged**